tests/functional/common.config

```diff
@@ -72,6 +72,11 @@ export SED_PROG="`set_prog_path sed`"
 export BC_PROG="`set_prog_path bc`"
 [ "$BC_PROG" = "" ] && _fatal "bc not found"
 
+PGREP_OPTIONS='-a'
+if pgrep * -a 2>&1 | grep -q "invalid option";then
```

Comment:
Thanks. It's my mistake. Another commit was committed to fix it. 
But I have no idea what is the better to handle this. Open a new PR, or just did as what I did now, or else? In the email way, we can resend a patch with a prefix "patch v2" and only "patch v2" was merged.